core/services/feeds/service.go

```diff
@@ -1128,6 +1180,24 @@ func (s *service) generateJob(ctx context.Context, spec string) (*job.Job, error
 		js, err = ocrbootstrap.ValidatedBootstrapSpecToml(spec)
 	case job.FluxMonitor:
 		js, err = fluxmonitorv2.ValidatedFluxMonitorSpec(s.jobCfg, spec)
+	case job.Workflow:
+		// TODO what the right way to validate a workflow spec?
```

Comment:
@HenryNguyen5 to confirm

Reply:
This needs to be merged, handling it after CRIB https://github.com/smartcontractkit/chainlink/pull/13125

Reply:
what is the eta for that? this feature is largely independent of validation details and needs to be merged for the CLO timeline of EOM. said differently, we can update the validation logic after this if we need to change it